controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -18,6 +18,13 @@
 GAME_DURATION_SECONDS = 150
 
 
+def log_filename() -> str:
+    # TODO: maybe merge with `recording_path`?
```

Comment:
Note: I'm not doing this here because we expect that #181 will change the way that `recording_path` works.